include/votca/csg/bead.h

```diff
@@ -308,49 +312,49 @@ class Bead : public BaseBead {
    *
    * \return Property object containing options
    */
-  Property &Options() { return *_options; }
+  Property &Options() { return *options_; }
 
   /**
    * update pointer to options object of bead
    * \param options pointer to options object of bead
    */
-  void setOptions(Property &options) { _options = &options; }
+  void setOptions(Property &options) { options_ = &options; }
 
-protected:
-  std::vector<int> _parent_beads;
+ protected:
+  std::vector<int> parent_beads_;
 
   // TODO: this is so far a pointer. this should change! each bead should have
   // own options.
-  Property *_options;
+  Property *options_;
 
-  byte_t _symmetry;
-  double _q;
+  byte_t symmetry_;
+  double charge_;
 
-  int _resnr;
+  int residue_number_;
 
-  vec _vel, _f, _u, _v, _w;
+  vec velocity_, bead_force_, u_, v_, w_;
 
-  bool _bVel;
-  bool _bU;
-  bool _bV;
-  bool _bW;
-  bool _bF;
+  bool bead_velocity_set_;
+  bool bU_;
+  bool bV_;
+  bool bW_;
+  bool bead_force_set_;
 
   /// constructur
-  Bead(Topology *owner, int id, std::weak_ptr<BeadType> type, byte_t symmetry, std::string name,
-       int resnr, double m, double q)
-      : _symmetry(symmetry), _q(q), _resnr(resnr) {
-    _parent = owner;
+  Bead(Topology *owner, int id, std::weak_ptr<BeadType> type, byte_t symmetry,
+       std::string name, int resnr, double m, double q)
+      : symmetry_(symmetry), charge_(q), residue_number_(resnr) {
+    topology_item_._parent = owner;
     setId(id);
     setType(type);
     setName(name);
     setMass(m);
-    _bPos = false;
-    _bVel = false;
-    _bU = false;
-    _bV = false;
-    _bW = false;
-    _bF = false;
+    bead_position_set_ = false;
+    bead_velocity_set_ = false;
+    bU_ = false;
+    bV_ = false;
+    bW_ = false;
+    bead_force_set_ = false;
   }
 
   void *_userdata;
```

Comment:
where is this used? The type information has been cased away

Reply:
@junghans do you have an idea about this. I am not good with pointers

Reply:
I keep seeing this user data pointer in other classes but it still isn't clear to me what exactly they are used for.

Reply:
I can't find it anywhere so I guess I'll pull it out.

Reply:
The same is true for Options it does not appear to be used anywhere...